helm_deploy/help-with-prison-visits-asynchronous-worker/templates/daily-tasks-cron.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ include "app.fullname" . }}-cron-daily-tasks
+  labels:
+    {{- include "app.labels" . | nindent 4 }}
+spec:
+  schedule: "30 4 * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 5
+  startingDeadlineSeconds: 600
+  successfulJobsHistoryLimit: 5
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: daily-tasks
+              image: quay.io/hmpps/help-with-prison-visits-asynchronous-worker
```

Comment:
might want to a `command:` option here as well just to be clear what is going to run.